release/release.sh

```diff
@@ -1,21 +1,38 @@
 #!/bin/bash
 #
 # Usage:
-#   $ release.sh [-f] <organization/project-name>
+#   $ release.sh [-f] <organization/project-name> [new-version]
```

Comment:
s/new-version/<new-version>/ perhaps?

Reply:
Oops ... forgot to escape angled-brackets.
Should have read ... `s/new-version/<new-version>/`

Reply:
Done